tdrs-frontend/yarn.lock

```diff
@@ -4226,10 +4226,10 @@ classlist-polyfill@^1.0.3:
   resolved "https://registry.yarnpkg.com/classlist-polyfill/-/classlist-polyfill-1.2.0.tgz#935bc2dfd9458a876b279617514638bcaa964a2e"
   integrity sha1-k1vC39lFiodrJ5YXUUY4vKqWSi4=
 
-classnames@^2.2.6:
-  version "2.2.6"
-  resolved "https://registry.yarnpkg.com/classnames/-/classnames-2.2.6.tgz#43935bffdd291f326dad0a205309b38d00f650ce"
-  integrity sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q==
+classnames@^2.3.0:
```

Comment:
@jorgegonzalez is there a reason this is 2.3.0 but the version is 2.3.1?

Reply:
Due to [semantic versioning](https://docs.npmjs.com/about-semantic-versioning), `^2.3.0` will resolve to the latest patch version (`2.3.1` in this case) if it is available.